### PR TITLE
Change E2E testing Safe addresses

### PR DESCRIPTION
**Pull request by hectorgomezv**
## Changes
- Change the testing Safe address used in E2E tests to `0x8675B754342754A30A2AeF474D114d8460bca19b`

### CODE REVIEW

**Review by iamacook — COMMENTED** [review]
Nit: I think we should move these to a constants file. It could even be a mapping of Safe address to owners. What do you think?

**Comment by hectorgomezv** [comment]
> Nit: I think we should move these to a constants file. It could even be a mapping of Safe address to owners. What do you think?

I totally agree. I've added the data to a `constants.ts` file under `src/routes/common/__tests__`. Please let me know if you think it should be stored in another place. Also, I've included the mappings in an object as you suggested. If you agree with the approach, what do you think about including the `chainId` also in that object?

File: src/routes/common/__tests__/constants.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { getAddress } from 'viem';
+
+interface TestSafe {
+  address: `0x${string}`;
+  owners: `0x${string}`[];
+}
+
+export const TEST_SAFE: TestSafe = {
+  address: getAddress('0x8675B754342754A30A2AeF474D114d8460bca19b'),
+  owners: [getAddress('0x6c15f69EE76DA763e5b5DB6f7f0C29eb625bc9B7')],
```
Comment (iamacook) [inline src/routes/common/__tests__/constants.ts:10]: Setting these to `as const` should negate the need to checksum them.

```suggestion
interface TestSafe {
  address: `0x${string}`;
  owners: `0x${string}`[];
}

export const TEST_SAFE: TestSafe = {
  address: '0x8675B754342754A30A2AeF474D114d8460bca19b' as const,
  owners: ['0x6c15f69EE76DA763e5b5DB6f7f0C29eb625bc9B7' as const],
```

**Review by iamacook — COMMENTED** [review]
> I totally agree. I've added the data to a `constants.ts` file under `src/routes/common/__tests__`. Please let me know if you think it should be stored in another place. Also, I've included the mappings in an object as you suggested. If you agree with the approach, what do you think about including the `chainId` also in that object?

I think the `chainId` would be a good addition.

**Comment by hectorgomezv** [comment]
> I think the `chainId` would be a good addition.

Added in [1d74f14](https://github.com/safe-global/safe-client-gateway/pull/1620/commits/1d74f14c31a279faf27dc654b26cc29bdab2c713)

**Review by iamacook — APPROVED** [review]

**Pull request merged**